.github/workflows/build-grpc-smoke-dist.yml

```diff
@@ -27,7 +27,6 @@ jobs:
         uses: burrunan/gradle-cache-action@v1.10
         with:
           job-id: grpc-smoke
-          save-gradle-dependencies-cache: false
```

Comment:
It is confusingly named property, I have confirmed it with Vladimir. In this case of `build-smoke` I think we should use `read-only: true` instead